Common/Utilities.cs

```diff
@@ -2825,22 +2810,23 @@ public static void UploadFileToWebApp(IPublishingProfile profile, string filePat
                 }))
                 {
                     var fileinfo = new FileInfo(filePath);
-                    ftpClient.LoginAsync().GetAwaiter().GetResult();
-                    if (!ftpClient.ListDirectoriesAsync().GetAwaiter().GetResult().Any(fni => fni.Name == "site"))
+                    await ftpClient.LoginAsync();

```

Comment:
Still use `.ConfigureAwait(false)` - recommended in all libraries, though not necessarily in application code.

```csharp
await ftpClient.LoginAsync().ConfigureAwait(false);
```

Reply:
Resolved.